### ISSUE TITLE
enhancement for Autolog

### ISSUE THREAD

**Issue by SavexG** [lambda-client/lambda-legacy#89]
Autolog sometimes logs you if you fall down a far distance because of baritone accting strange or other 1 time damage occurrences
and you have to disable it or set it really really low if you want to connect again without getting killed 

this new mode should be so it only logs you out again after you reconnect when the health goes down further when you log in and it should act as normal when it surpasses the Min Health set 

this feature would be really god for automation so you don't have to manually sit there and wait to log in and see if your bots get killed or not and then set autolog again 

this would be specially useful for example Highwaywork near spawn where you get autloged a lot when running Highwaytools afk
and paired with autoeat

**Comment by WooHooASDF** [comment]
For clarification, when you get kicked, you want it to reconnect, and then re-enable auto log once you are at a higher health?
Or just lower the health required to log automatically.

**Comment by SavexG** [comment]
Yes like that so imagine you are walking afk on a Highway with baritone then someone sees you and explodes 1 crystal on you 
you log because your health goes below 10 (I set minhealth to 10)
then you relog and when the person there is still camping and explodes another crystal on you = health goes down or you pop a totem it will log you out again

if there is no threat anymore = health doesn't go down or pop a totem then it will not log you out 
this is preferably paired with autoeat so you always eat a gapple when health it down and then baritone keeps continuing as normal


tldr imagine you have minhealth set 10

you log out because health goes below 10
if you relog and health goes down further or totem pop you log out again
else it doesn't log you out and you should eat a gapple

**Comment by SavexG** [comment]
please ask again if there are more uncertainties I tried my best but I know its still a little confusingly explained

**Comment by WooHooASDF** [comment]
I’m not actually a dev (I don’t know Kotlin), I just found that initial message a little confusing and decided to help clear it up.
